frontend/src/Routes/routes.tsx

```diff
@@ -6,21 +6,21 @@ import SignUp from "../Pages/auth/SignupPage";
 import AdminLogin from "../Pages/auth/AdminLogin";
 import AdminSignin from "../Pages/auth/AdminSignUp";
 import Dashboard from "../Pages/Dashboard";
-
+import AddWordPage from "../Pages/AddWordPage";
 
 const Main = () => {
-  return(
+  return (
     <Routes>
-      <Route path="/" element={<HomePage/>} />
-      <Route path="/categories" element={<CategoryPage/>} />
-      <Route path="/login" element={<LoginPage/>} />
-      <Route path="/register" element={<SignUp/>} />
-      <Route path="/admin-login" element={<AdminLogin/>} />
-      <Route path="/admin-register" element={<AdminSignin/>} />
-      <Route path="/dashboard" element={<Dashboard/>} />
+      <Route path="/" element={<HomePage />} />
+      <Route path="/categories" element={<CategoryPage />} />
+      <Route path="/login" element={<LoginPage />} />
+      <Route path="/register" element={<SignUp />} />
+      <Route path="/admin-login" element={<AdminLogin />} />
+      <Route path="/admin-register" element={<AdminSignin />} />
+      <Route path="/dashboard" element={<Dashboard />} />
+      <Route path="admin/category/addword" element={<AddWordPage />} />
```

Comment:
for the route path , it is better to use this path `admin/category/{category_id}/add`, same we used path in backend.